pkg/runctl/controller.go

```diff
@@ -414,7 +414,7 @@ func (c *Controller) syncHandler(key string) error {
 		}
 		started := run.GetStartTime()
 		if started != nil {
-			if err = c.changeState(pipelineRun, api.StateRunning); err != nil {
+			if err = c.changeState(pipelineRun, api.StateRunning, metav1.Now()); err != nil {
```

Comment:
I think we could also get the Running Start Time from tekton here.

Reply:
done